test/integration/host_js_test.rb

```diff
@@ -22,6 +22,34 @@ class HostJSTest < IntegrationTestWithJavascript
     SETTINGS[:organizations_enabled] = true
   end
 
+  describe 'multiple hosts selection' do
+    setup do
+      FactoryBot.create(:host)
+      FactoryBot.create(:host)
+    end
+
+    test 'hosts counter should refer to per_page value first (max prespective)' do
+      Setting[:entries_per_page] = 2
```

Comment:
This needs to be reset after every test to the original value, or you may have unexpected results in other tests.